tests/util/transactions.ts

```diff
@@ -432,3 +434,25 @@ export const sendAllStreamAndWaitLast = async (
   }
   await Promise.all(promises);
 };
+
+export const ERC20_TOTAL_SUPPLY = 1_000_000_000n;
+export const setupErc20Contract = async (context: DevTestContext, name: string, symbol: string) => {
+  const { contract, contractAddress, rawTx } = await createContract(
+    context,
+    "ERC20WithInitialSupply",
+    {
+      ...ALITH_TRANSACTION_TEMPLATE,
+      gas: 5_000_000,
+    },
+    [name, symbol, ALITH_ADDRESS, ERC20_TOTAL_SUPPLY]
+  );
+  const { result } = await context.createBlock(rawTx);
+  expectEVMResult(result.events, "Succeed");
+  return { contract, contractAddress };
+};
+
+export async function getBalance(context: DevTestContext, blockHeight: number, address: string) {
```

Comment:
This function seems useless, we already have `context.web3.eth.getBalance`

Reply:
This function was not added by me, it was already present inside `test-xcm-erc20-transfer.ts` file. It is used there, but I can remove it and refactor that file as well.

Reply:
> This function was not added by me

I know, but we have a lot of things to improve in the existing codebase, so at each modification we take the opportunity to improve the code at the modified place :)

Reply:
Totally agree, I just pushed the changes :)